packages/haul-ram-bundle-webpack-plugin/package.json

```diff
@@ -12,11 +12,11 @@
   "license": "MIT",
   "bugs": "https://github.com/callstack/haul/issues",
   "peerDependencies": {
-    "@haul-bundler/core": "^0.10.0",
-    "metro": "^0.53.1",
-    "terser": "^3.17.0",
-    "webpack": "^4.30.0",
-    "webpack-sources": "^1.3.0"
+    "@haul-bundler/core": "0.10.0",
+    "metro": ">= 0.53.1",
```

Comment:
quite permissive, maybe worth to set a top limit to 0.55?

Reply:
Technically we should make it `0.51.1` and above since this one ships with RN 0.59

Reply:
@thymikee WDYT about `>=0.51.1 < 1`?

Reply:
depends on the APIs Haul use and how stable are they, but if they sit there for a while, I'm good

Reply:
Well we've been using `RamBundleParser` from `0.51.1` and newer without any issues.